packages/pigeon/example/lib/main.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Example plugin for pigeon code excerpts and examples.
+class ExamplePluginClass {
```

Comment:
This file isn't used for anything. It just needs to exist so that code-excerpts can run. This code was used for a plugin I made and deleted. Will clear it out.